frontend/public/components/operator-lifecycle-manager/package-manifest.tsx

```diff
@@ -55,34 +56,37 @@ export const PackageManifestList: React.SFC<PackageManifestListProps> = (props)
-    ? <React.Fragment>
-      { _.isEmpty(props.data) && <MsgBox title="No Package Manifests Found" detail="Package Manifests are packaged Operators which can be subscribed to for automatic upgrades." /> }
-      { [...catalogs.values()].map(catalog => <div key={catalog.name} className="co-catalogsource-list__section">
-        <div className="co-catalogsource-list__section__packages">
-          <div>
-            <h3>{catalog.displayName}</h3>
-            <span className="text-muted">Packaged by {catalog.publisher}</span>
-          </div>
-          <Link to={`/k8s/ns/${catalog.namespace}/${referenceForModel(CatalogSourceModel)}/${catalog.name}`}>View catalog details</Link>
+  return <StatusBox
+    loaded={props.loaded}
+    loadError={props.loadError}
+    label={PackageManifestModel.labelPlural}
+    data={props.data}
+    EmptyMsg={() => <MsgBox title="No Package Manifests Found" detail="Package Manifests are packaged Operators which can be subscribed to for automatic upgrades." />}>
+    { [...catalogs.values()].map(catalog => <div key={catalog.name} className="co-catalogsource-list__section">
+      <div className="co-catalogsource-list__section__packages">
+        <div>
+          <h3>{catalog.displayName}</h3>
+          <span className="text-muted">Packaged by {catalog.publisher}</span>
         </div>
-        <List
-          loaded={true}
-          data={props.data.filter(pkg => pkg.status.catalogSource === catalog.name)}
-          filters={props.filters}
-          virtualize={false}
-          Header={PackageManifestHeader}
-          Row={(rowProps: {obj: PackageManifestKind}) => <PackageManifestRow
-            obj={rowProps.obj}
-            catalogSourceName={catalog.name}
-            catalogSourceNamespace={catalog.namespace}
-            subscription={props.subscription.data.find(sub => sub.spec.name === rowProps.obj.metadata.name)} />}
-          label="Package Manifests"
-          EmptyMsg={() => <MsgBox title="No PackageManifests Found" detail="The catalog author has not added any packages." />} />
-      </div>) }
-    </React.Fragment>
-    : <StatusBox loaded={props.loaded} loadError={props.loadError} label={PackageManifestModel.labelPlural} />;
-};
+        <Link to={`/k8s/ns/${catalog.namespace}/${referenceForModel(CatalogSourceModel)}/${catalog.name}`}>View catalog details</Link>
+      </div>
+      <List
+        loaded={true}
+        data={props.data.filter(pkg => pkg.status.catalogSource === catalog.name)}
+        filters={props.filters}
+        virtualize={false}
+        Header={PackageManifestHeader}
+        Row={(rowProps: {obj: PackageManifestKind}) => <PackageManifestRow
+          obj={rowProps.obj}
+          catalogSourceName={catalog.name}
+          catalogSourceNamespace={catalog.namespace}
+          subscription={props.subscription.data.find(sub => sub.spec.name === rowProps.obj.metadata.name)}
+          defaultNS={_.get(props.operatorGroup.data, '[0].metadata.namespace')} />}
```

Comment:
Direct user to the first namespace with an `OperatorGroup` when viewing "All Namespaces" and clicking "Create Subscription".